react_projects/rick-and-morty-web/ToDo.md

```diff
@@ -0,0 +1,14 @@
+# To-Do list:
+# Functionalities
+- [ ] Pagination view:
+    - [x] include last pages && use (...) in the middle
+    - [x] reload paginator after the search to adapt the new list results
+- [ ] Set pageNumber in 1 after a new search
+- [ ] Not Found error
+- [ ] Add more filters
+- [ ] Manage more endpoints
```

Comment:
I also recommend giving a read to https://github.com/xmartlabs/react-template-xmartlabs?tab=readme-ov-file#networking to understand how we isolate API management from components